### PR TITLE
Add iron query params

### PR DESCRIPTION
**Pull request by rictic**
This has been working well for all of our demos, tests, etc, let's merge it into master!

### CODE REVIEW

**Comment by abdonrd** [comment]
This branch needs a rebase. :)

File: demo/index.html [hunk 1]
```diff
@@ -87,21 +92,56 @@
           </paper-slider>
         </div>
       </div>
+      <div layout horizontal flex>
```
Comment (rictic) [inline demo/index.html]: This part of the demo was half baked and probably should be deleted for now

File: demo/iron-query-params.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (rictic) [inline demo/iron-query-params.html]: `@license`

File: test/iron-query-params.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<!doctype html>
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (rictic) [inline test/iron-query-params.html]: `@license`

**Comment by rictic** [comment]
Ok, fixed all of the things that I was just complaining about me having written some time ago.

File: index.html [hunk 4]
```diff
@@ -1,11 +1,12 @@
 <!doctype html>
 <!--
+@license
 Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline index.html]: License should probably also be updated to `2016`

File: demo/index.html [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <!--
 @license
-Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline demo/index.html]: Derp.

File: test/iron-page-url.html [hunk 6]
```diff
@@ -46,27 +46,6 @@
     });
   }
 
-  function tryUntilTrue(f) {
```
Comment (cdata) [inline test/iron-page-url.html]: :+1:

**Comment by cdata** [comment]
Small license detail, otherwise LGTM :+1:

File: index.html (hunk 4 above)
Comment (rictic) [inline index.html]: My understanding of the policy is that we set the license at file creation time, so I'm keeping the dates in place.

**Pull request merged**